### PR TITLE
ISO: persistently mount /var/lib/containerd

### PR DESCRIPTION
**Pull request by medyagh**
before this PR:
/var/lib/containerd was stored in tmpfs

```
$  grep /var /proc/mounts | egrep -v "^(overlay|shm)"
/dev/vda1 /var/lib/boot2docker ext4 rw,relatime 0 0
/dev/vda1 /var/lib/docker ext4 rw,relatime 0 0
/dev/vda1 /var/lib/containers ext4 rw,relatime 0 0
/dev/vda1 /var/log ext4 rw,relatime 0 0
/dev/vda1 /var/lib/kubelet ext4 rw,relatime 0 0
/dev/vda1 /var/lib/cni ext4 rw,relatime 0 0
/dev/vda1 /var/lib/minikube ext4 rw,relatime 0 0
/dev/vda1 /var/lib/toolbox ext4 rw,relatime 0 0
/dev/vda1 /var/lib/minishift ext4 rw,relatime 0 0
tmpfs /var/lib/kubelet/pods/6a9c4fa6-7027-48db-8f80-1ecb3c6e568b/volumes/kubernetes.io~secret/storage-provisioner-token-th5bp tmpfs rw,relatime 0 0
tmpfs /mnt/vda1/var/lib/kubelet/pods/6a9c4fa6-7027-48db-8f80-1ecb3c6e568b/volumes/kubernetes.io~secret/storage-provisioner-token-th5bp tmpfs rw,relatime 0 0
tmpfs /var/lib/kubelet/pods/615c7df5-e19c-440c-8688-000febf34ade/volumes/kubernetes.io~secret/kube-proxy-token-gscfv tmpfs rw,relatime 0 0
tmpfs /mnt/vda1/var/lib/kubelet/pods/615c7df5-e19c-440c-8688-000febf34ade/volumes/kubernetes.io~secret/kube-proxy-token-gscfv tmpfs rw,relatime 0 0
tmpfs /var/lib/kubelet/pods/1bc7a49f-4420-4e8e-a9d1-f2d100558136/volumes/kubernetes.io~secret/coredns-token-mbzzs tmpfs rw,relatime 0 0
tmpfs /mnt/vda1/var/lib/kubelet/pods/1bc7a49f-4420-4e8e-a9d1-f2d100558136/volumes/kubernetes.io~secret/coredns-token-mbzzs tmpfs rw,relatime 0 0
tmpfs /var/lib/kubelet/pods/b96c5a5a-5e2c-4373-a481-2a4a571a936e/volumes/kubernetes.io~secret/coredns-token-mbzzs tmpfs rw,relatime 0 0
tmpfs /mnt/vda1/var/lib/kubelet/pods/b96c5a5a-5e2c-4373-a481-2a4a571a936e/volumes/kubernetes.io~secret/coredns-token-mbzzs tmpfs rw,relatime 0 0
```

## After this PR
```
$ grep /var /proc/mounts | egrep -v "^(overlay|shm)"
/dev/vda1 /var/lib/boot2docker ext4 rw,relatime 0 0
/dev/vda1 /var/lib/docker ext4 rw,relatime 0 0
/dev/vda1 /var/lib/containerd ext4 rw,relatime 0 0
/dev/vda1 /var/lib/containers ext4 rw,relatime 0 0
/dev/vda1 /var/log ext4 rw,relatime 0 0
/dev/vda1 /var/tmp ext4 rw,relatime 0 0
/dev/vda1 /var/lib/kubelet ext4 rw,relatime 0 0
/dev/vda1 /var/lib/kubelet/pods ext4 rw,relatime 0 0
/dev/vda1 /mnt/vda1/var/lib/kubelet/pods ext4 rw,relatime 0 0
/dev/vda1 /var/lib/cni ext4 rw,relatime 0 0
/dev/vda1 /var/lib/minikube ext4 rw,relatime 0 0
/dev/vda1 /var/lib/toolbox ext4 rw,relatime 0 0
/dev/vda1 /var/lib/minishift ext4 rw,relatime 0 0
tmpfs /var/lib/kubelet/pods/e498f978-13ca-4561-bd82-f729a79e0b22/volumes/kubernetes.io~secret/kube-proxy-token-5fb45 tmpfs rw,relatime 0 0
tmpfs /mnt/vda1/var/lib/kubelet/e498f978-13ca-4561-bd82-f729a79e0b22/volumes/kubernetes.io~secret/kube-proxy-token-5fb45 tmpfs rw,relatime 0 0
tmpfs /var/lib/kubelet/e498f978-13ca-4561-bd82-f729a79e0b22/volumes/kubernetes.io~secret/kube-proxy-token-5fb45 tmpfs rw,relatime 0 0
tmpfs /mnt/vda1/var/lib/kubelet/pods/e498f978-13ca-4561-bd82-f729a79e0b22/volumes/kubernetes.io~secret/kube-proxy-token-5fb45 tmpfs rw,relatime 0 0
tmpfs /var/lib/kubelet/pods/76633a80-973a-4b68-9f5e-f39d4c781709/volumes/kubernetes.io~secret/coredns-token-qcxrj tmpfs rw,relatime 0 0
tmpfs /mnt/vda1/var/lib/kubelet/76633a80-973a-4b68-9f5e-f39d4c781709/volumes/kubernetes.io~secret/coredns-token-qcxrj tmpfs rw,relatime 0 0
tmpfs /var/lib/kubelet/76633a80-973a-4b68-9f5e-f39d4c781709/volumes/kubernetes.io~secret/coredns-token-qcxrj tmpfs rw,relatime 0 0
tmpfs /mnt/vda1/var/lib/kubelet/pods/76633a80-973a-4b68-9f5e-f39d4c781709/volumes/kubernetes.io~secret/coredns-token-qcxrj tmpfs rw,relatime 0 0
tmpfs /var/lib/kubelet/pods/ce976354-8acc-4f4c-b63a-9c7bef5ad986/volumes/kubernetes.io~secret/coredns-token-qcxrj tmpfs rw,relatime 0 0
tmpfs /mnt/vda1/var/lib/kubelet/ce976354-8acc-4f4c-b63a-9c7bef5ad986/volumes/kubernetes.io~secret/coredns-token-qcxrj tmpfs rw,relatime 0 0
tmpfs /var/lib/kubelet/ce976354-8acc-4f4c-b63a-9c7bef5ad986/volumes/kubernetes.io~secret/coredns-token-qcxrj tmpfs rw,relatime 0 0
tmpfs /mnt/vda1/var/lib/kubelet/pods/ce976354-8acc-4f4c-b63a-9c7bef5ad986/volumes/kubernetes.io~secret/coredns-token-qcxrj tmpfs rw,relatime 0 0
tmpfs /var/lib/kubelet/pods/bfd58ce3-5687-4b13-bd10-ece1ff1d9dcb/volumes/kubernetes.io~secret/storage-provisioner-token-zzsbh tmpfs rw,relatime 0 0
tmpfs /mnt/vda1/var/lib/kubelet/bfd58ce3-5687-4b13-bd10-ece1ff1d9dcb/volumes/kubernetes.io~secret/storage-provisioner-token-zzsbh tmpfs rw,relatime 0 0
tmpfs /var/lib/kubelet/bfd58ce3-5687-4b13-bd10-ece1ff1d9dcb/volumes/kubernetes.io~secret/storage-provisioner-token-zzsbh tmpfs rw,relatime 0 0
tmpfs /mnt/vda1/var/lib/kubelet/pods/bfd58ce3-5687-4b13-bd10-ece1ff1d9dcb/volumes/kubernetes.io~secret/storage-provisioner-token-zzsbh tmpfs rw,relatime 0 0
```
this PR will add /var/lib/containerd to disk instead of tmpfs that caused preload containerd to fail
and possible source of other mysterious containerd test failures.  

also put /kubelet/pods there too since that was only folder on tempfs before this PR. leaving the rest of /var to be presitant in following up PRs

### CODE REVIEW

**Comment by afbjorklund** [comment]
Would be a less intrusive change to only move `/var/lib/containerd`, but

**Comment by afbjorklund** [comment]
This needs a rebase, without the podman revert

File: deploy/iso/minikube-iso/package/podman/podman.mk [hunk 1]
```diff
@@ -29,8 +29,6 @@ endef
 
 define PODMAN_INSTALL_TARGET_CMDS
 	$(INSTALL) -Dm755 $(@D)/bin/podman $(TARGET_DIR)/usr/bin/podman
-	$(INSTALL) -d -m 755 $(TARGET_DIR)/etc/cni/net.d/
-	$(INSTALL) -m 644 cni/87-podman-bridge.conflist $(TARGET_DIR)/etc/cni/net.d/87-podman-bridge.conflist
```
Comment (afbjorklund) [inline deploy/iso/minikube-iso/package/podman/podman.mk:33]: this seems unrelated to /var and containerd

Comment (medyagh) [reply deploy/iso/minikube-iso/package/podman/podman.mk:33]: sounds good,  I will revert that once the other PR is merged, I added it here only so I can test it with the other PR( since without it it was failing)

**Comment by medyagh** [comment]
once jenkins is done the URL will be https://storage.cloud.google.com/minikube-builds/7843/minikube.iso

**Comment by medyagh** [comment]
> Would be a less intrusive change to only move `/var/lib/containerd`, but

I thought the same but  that was thomas idea that this is technical debt and needs to be addreessed in the same PR

**Comment by afbjorklund** [comment]
I think you should leave re-organizing `/var` to the next release, and do a simple fix like in #7740

File: deploy/iso/minikube-iso/package/automount/minikube-automount [hunk 2]
```diff
@@ -117,6 +121,10 @@ if [ -n "$BOOT2DOCKER_DATA" ]; then
     mkdir /var/lib/kubelet
     mount --bind /mnt/$PARTNAME/var/lib/kubelet /var/lib/kubelet
 
+    mkdir -p /mnt/$PARTNAME/var/lib/kubelet/pods
```
Comment (tstromberg) [inline deploy/iso/minikube-iso/package/automount/minikube-automount:124]: I don't think adding /var/lib/kubelet adds value to what you are trying to fix. I'd just remove it.

Comment (medyagh) [reply deploy/iso/minikube-iso/package/automount/minikube-automount]: done

File: deploy/iso/minikube-iso/package/podman/podman.mk (hunk 1 above)
Comment (medyagh) [reply deploy/iso/minikube-iso/package/podman/podman.mk]: done
